frontend/src/router.js

```diff
@@ -7,16 +7,18 @@ const HelloWorld = () => import("./pages/HelloWorld.vue");
 const Screening = () => import("./pages/Screening.vue");
 const Matching = () => import("./pages/Matching.vue");
 const StudentProfile = () => import("./pages/StudentProfile.vue");
+const EditStudent = () => import("./pages/EditStudent.vue")
 const NewStudent = () => import("./pages/NewStudent.vue");
 
 const router = new VueRouter({
   mode: "history",
   hash: false,
   routes: [
-    { path: "/", component: HelloWorld },
+    { path: "/", component: HelloWorld},
     { path: "/screening", component: Screening },
     { path: "/matching", component: Matching },
     { path: "/student/:id", component: StudentProfile },
+    { path: "/student/:id/edit", component: EditStudent },
```

Comment:
@aelbore Not sure how vue router resolves this - can we have both `/student/:id` and `/student/:id/edit`? Or will it always try to match the first one?

Reply:
hi thomas!! sorry for the late reply. i'll try answering this. 
I think it should work because the two routes won't be accessed at the same level/time (not sure what the right term is) — /edit can only be accessed after /student/:id is accessed, because the "edit" button only appears on each student's profile. [Figma: Student Profile](https://www.figma.com/file/4QJfRsrEGsM3h9qr9jejt0/IRR-Hi-fi?node-id=0%3A1 
) 
So I'm thinking that the flow goes like this: All Students (table view) → click on individual students → access student profile (/student/:id) → click "edit" → Edit Student (/student/:id/edit)

The [docs for dynamic route matching](https://router.vuejs.org/guide/essentials/dynamic-matching.html) seem to allow for this. 
This is the example they give, which supports even having multiple dynamic segments on one route:

pattern | matched path | $route.params
-- | -- | --
/user/:username | /user/evan | { username: 'evan' }
/user/:username/post/:post_id | /user/evan/post/123 | { username: 'evan', post_id: '123' }

Reply:
update: somehow the routes don't resolve so the editstudent route is just /edit for now.

Reply:
update: routes work now, just needed to add <base href ="/" .> to index.html